packages/client/lib/sync/sync.ts

```diff
@@ -100,7 +101,9 @@ export abstract class Synchronizer extends EventEmitter {
     }, this.interval * 30)
     while (this.running) {
       try {
-        if (await this.sync()) this.emit('synchronized')
+        if (await this.sync()) {
+          this.config.events.emit(Event.SYNC_SYNCHRONIZED)
```

Comment:
@ryanio What is this event doing?  The name seems to indicate it's emitted when the chain is fully synced and we're at the tip, but I didn't think we had anything implemented around this and what we're putting in the logs is just "Sychronized" without saying what was synchronized.

Reply:
Yep the `sync()` promise resolves when finished syncing with the peer. For example see the small blockchains set up and synced in the integration tests.

Reply:
What do you think about adding some additional detail to the log message, maybe something like `Synchronized chain with best peer: ${peer.address} at height ${this.chain.blocks.height`, to give clarity to this event?

Reply:
yeah that's a good idea!